Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -4418,6 +4435,19 @@ void RenderBox::computePositionedLogicalHeight(LogicalExtentComputedValues& comp
     Length logicalTopLength = styleToUse.logicalTop();
     Length logicalBottomLength = styleToUse.logicalBottom();
 
+    // https://drafts.csswg.org/css-anchor-position-1/#anchor-center
+    bool alignsToAnchorCenter = container()->isHorizontalWritingMode() != isHorizontalWritingMode()
+        ? style().justifySelf().position() == ItemPosition::AnchorCenter
+        : style().alignSelf().position() == ItemPosition::AnchorCenter;
+    RenderBoxModelObject* defaultAnchorBox = alignsToAnchorCenter ? dynamicDowncast<RenderBoxModelObject>(defaultAnchorRenderer()) : nullptr;
```

Comment:
```suggestion
    CheckedPtr defaultAnchorBox = alignsToAnchorCenter ? dynamicDowncast<RenderBoxModelObject>(defaultAnchorRenderer()) : nullptr;
```

Reply:
`defaultAnchorBox` can be `nullptr` in the above, so `CheckedPtr` will trigger assertion, won't it?

Reply:
CheckedPtr as the name implies can hold a nullptr value (look at its c'tors) Maybe it's CheckedRef you were thinking of?

Reply:
unconditionally derefing it may trigger assertion.

Reply:
Right, I confused it with `CheckedRef` - sorry for confusion.